### DISCUSSION TITLE
Schlechte Bild Qualität | Poor image quality

### DISCUSSION THREAD

**Discussion by nhet** [jomjol/AI-on-the-edge-device#2833] [Q&A]
Hallo zusammen,
ich habe das Problem, dass Bild mit Streifen durchzogen ist. Ich kann mich an eine Anleitung erinnern, um diese zu entfernen / zu umgehen. Leider finde ich da nix mehr.
Ich habe bereits ein neues Netzteil ausprobiert und ein Kondensator zur Stabilisierung eingesetzt.

--English
Hello everyone,
I have the problem that the picture is streaked with stripes. I can remember instructions on how to remove / bypass them. Unfortunately I can't find any more.
I have already tried a new power supply and used a capacitor for stabilization.

--Image
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/13017349/2aa425e0-bd2f-462c-8f8b-5efd0ea4aca2)

**Comment by arkoode** [comment]
Kann es sein das die ESP32-Cam defekt ist? Kannst du die außerhalb von ai testen?

**Reply by nhet** [reply]
Moin, 
leider nicht. Im Liveview laufen die Streifen teilweise von unten nach oben.
